tests/aws/services/lambda_/functions/lambda_error_responses.py

```diff
@@ -0,0 +1,7 @@
+def handler(event, context):
+    body = event.get("body")
+    status_code = body.get("httpStatus")
+    if int(status_code) >= 400:
+        return {"statusCode": 200, "body": "customerror"}
```

Comment:
From a Lambda perspective, these are not really "errors" but rather different outputs.

FYI: We added dedicated error testing in `tests.aws.services.lambda_.test_lambda.TestLambdaErrors` with scenarios covering both handler errors and runtime startup errors.

Reply:
Yep, the result of the call as in the HTTP status code is always 200. This example is just an example of mapping responses after the lambda call to "error" HTTP codes. 

Your comment is because of the lambda module name? Nevertheless this is not a lambda concern but API interpretation of a lambda response for expected errors, for example the input doesn't match an expectation therefore the lambda response contains a signal to be mapped by the gateway as a error (400 status code).

Reply:
Agree, that makes sense. (just a minor naming nit but feel free to ignore)

Testing actual Lambda errors is another scenario (maybe worth covering in another test).